ts/Series/Item/ItemSeries.ts

```diff
@@ -311,7 +311,8 @@ class ItemSeries extends PieSeries {
                         x: x,
                         y: y,
                         width: width,
-                        height: height
+                        height: height,
+                        fill: point.color // Circles color update (#17257)
```

Comment:
This works for this case, but doesn't work for changes in the stroke color, stroke width etc. In short, those properties returned by `pointAttribs`. These attributes are only applied to new graphics:

https://github.com/highcharts/highcharts/blob/a905c72df5599de95cf9a8fd040d59338dde530c/ts/Series/Item/ItemSeries.ts#L326-L328

So we need to change that so they are applied to existing graphics as well.